app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "About") %>
```

Comment:
Chú tìm hiểu `i18n in rails` và sử dụng cho toàn bộ file html nhé

tag `a` thì thay thành `link_to ` luôn nè